pkg/api/customization/oci/handler.go

```diff
@@ -0,0 +1,238 @@
+package oci
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/rancher/norman/api/access"
+	"github.com/rancher/norman/types"
+	client "github.com/rancher/types/client/management/v3"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/gorilla/mux"
+	"github.com/oracle/oci-go-sdk/common"
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/rancher/pkg/auth/util"
+	"github.com/rancher/rancher/pkg/ref"
+	v1 "github.com/rancher/types/apis/core/v1"
+	"github.com/rancher/types/apis/management.cattle.io/v3/schema"
+	"github.com/rancher/types/config"
+	"github.com/rancher/types/namespace"
+
+	"encoding/json"
+)
```

Comment:
```suggestion
import (
	"encoding/json"
	"fmt"
	"io/ioutil"
	"net/http"

	"github.com/gorilla/mux"
	"github.com/oracle/oci-go-sdk/common"
	"github.com/rancher/norman/api/access"
	"github.com/rancher/norman/httperror"
	"github.com/rancher/norman/types"
	"github.com/rancher/rancher/pkg/auth/util"
	"github.com/rancher/rancher/pkg/ref"
	v1 "github.com/rancher/types/apis/core/v1"
	"github.com/rancher/types/apis/management.cattle.io/v3/schema"
	client "github.com/rancher/types/client/management/v3"
	"github.com/rancher/types/config"
	"github.com/rancher/types/namespace"
	"github.com/sirupsen/logrus"
)
```

Reply:
If the spaces are deleted then goimports is ran, the output should look like this. Our normal style is two groups, standard library and everything else.

Reply:
I ran `goimports`. The CI job is also not complaining. Is there a particular version or flags I should be using?

```
jemillan-mac:rancher jemillan$ goimports -d pkg/api/customization/oci/handler.go  
jemillan-mac:rancher jemillan$ 
```